proxy-node-shared/src/test/java/uk/gov/ida/notification/contracts/verifyserviceprovider/TranslatedHubResponseTestDataHelper.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.ida.notification.contracts.verifyserviceprovider;
+
+import java.util.Collections;
+
+import org.joda.time.DateTime;
+
+public class TranslatedHubResponseTestDataHelper {
```

Comment:
EidasResponseTestDataGenerator could be a better name? Is eIDASResponse a better name for the TranslatedHubResponse Object too? I guess we only care that we are receiving a eIDAS response back from the VSP.

Reply:
I guess the eIDAS response hasn't been generated until it has reached the Translator though.